Sources/NIOEmbedded/Embedded.swift

```diff
@@ -97,6 +97,8 @@ public final class EmbeddedEventLoop: EventLoop {
     // execution.
     private var taskNumber: UInt64 = 0
 
+    public let description = "EmbeddedEventLoop"
```

Comment:
This should maybe include the pointer value of `self`? Just so we can distinguish them